common/djangoapps/student/views.py

```diff
@@ -141,6 +141,7 @@
 ReverifyInfo = namedtuple('ReverifyInfo', 'course_id course_name course_number date status display')  # pylint: disable=invalid-name
 SETTING_CHANGE_INITIATED = 'edx.user.settings.change_initiated'
 
+MAX_FAILED_LOGIN_ATTEMPTS_LOCKOUT_PERIOD_OVER_ESTIMATE_MINUTES=int(settings.MAX_FAILED_LOGIN_ATTEMPTS_LOCKOUT_PERIOD_SECS*.0222+1) #0.0222 is 33% more with the conversion into minutes.  1.33/60=0.02216
```

Comment:
This approach was really clever and I appreciate that it was your initial instinct. The more I think about it, this really should be a case where simple trumps clever. While the 30% calculation provides the expected outcome given this input (15min), it starts to fall apart if we ever change the timeout.

If we drop the timeout to 10m, the message will say 14.
If we raise the timeout to 20m, the message will say 27.

We wouldn't want to display an error message with odd numbers like that; we want something divisible by 5.
```python
DISPLAY_LOGIN_LOCKOUT_MINUTES = 5 + settings.MAX_FAILED_LOGIN_ATTEMPS_LOCKOUT_PERIOD_SECS
```

If we bump the timeout to 60m, we're adding 20 minutes to the wait time, instead of just 5.

Really, the intent here is to grant us just a little bit of leeway, in case they click even a few seconds early. 

5 minutes all the time should be plenty.